tensorly/backend/core.py

```diff
@@ -48,10 +48,6 @@ def float64(self):
     def float32(self):
         raise NotImplementedError
 
-    @property
-    def SVD_FUNS(self):
```

Comment:
SVD_FUNS is used in the tests.

Reply:
`SVD_FUNS` is still included in every backend. The change here is that core provides an implementation of it, which is then optionally overwritten in backends that provide a third SVD method. I thought this might be cleaner given that the majority of backends provide the same set?